types/test.ts

```diff
@@ -311,3 +311,23 @@ const lpWithdraw = StellarSdk.xdr.LiquidityPoolWithdrawOp.fromXDR(
   'base64'
 );
 lpWithdraw; // $ExpectType LiquidityPoolWithdrawOp
+
+const pubkey = masterKey.rawPublicKey(); // $ExpectType Buffer
+const seckey = masterKey.rawSecretKey(); // $ExpectType Buffer
+const muxed = StellarSdk.encodeMuxedAccount(masterKey.publicKey(), '1'); // $ExpectType MuxedAccount
+const muxkey = muxed.toXDR("raw"); // $ExpectType Buffer
+
+let result = StellarSdk.StrKey.encodeEd25519PublicKey(pubkey);  // $ExpectType string
```

Comment:
Not sure I really get the point of `ExpectType`.. The type signature is `(Buffer) -> string`, so if it didn't return that it wouldn't compile. but /shrug, doesn't hurt anything.